### PR TITLE
fix(components/editabletext): design adjustments

### PR DESCRIPTION
**Pull request by ncomont**
**What is the problem this PR is trying to solve?**

 EditableText needs to be aligned with the guidelines

**What is the chosen solution to this problem?**

Follow https://jira.talendforge.org/browse/TUX-533

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/EditableText/EditableText.scss [hunk 1]
```diff
@@ -69,25 +73,44 @@ $tc-input-editable-text-input-weight: $form-base-font-weight !default;
...
 				background: lighten($scooter, 25%);
 			}
 		}
 
 		&-buttons {
 			display: flex;
+			position: absolute;
+			right: $padding-large;
+
+			display: flex;
```
Comment (jdescottes-talend) [inline packages/components/src/EditableText/EditableText.scss]: `display: flex;` is already declared and triggers an sasslint error.

Comment (charlesnguyen) [inline packages/components/src/EditableText/EditableText.scss]: Why do you need them here?  I tested without them and it is ok.
```
align-items: center;
justify-content: center;
height: 32px;
top: 0;
```

File: packages/components/src/EditableText/InlineForm.component.js [hunk 2]
```diff
@@ -89,29 +89,33 @@ class InlineForm extends React.Component {
...
 					<Action
-						name="action-cancel-title"
-						label={t('CANCEL_TOOLTIP', { defaultValue: 'Cancel' })}
-						icon="talend-cross"
-						onClick={this.onCancel}
+						type="submit"
+						name="action-submit-title"
+						label={t('SUBMIT_TOOLTIP', { defaultValue: 'Submit' })}
+						icon="talend-check"
+						onClick={noop}
```
Comment (jdescottes-talend) [inline packages/components/src/EditableText/InlineForm.component.js]: Why you can't just don't set the prop ?

File: output/components.sasslint.txt [hunk 3]
```diff
@@ -4,8 +4,11 @@
 src/Drawer/Drawer.scss
   14:4  warning  Statement must begin on a new line  brace-style
 
+src/EditableText/EditableText.scss
+  103:4  error  Duplicate properties are not allowed within a block  no-duplicate-properties
```
Comment (jsomsanith) [inline output/components.sasslint.txt]: You have 2 `display: flex;`

File: packages/components/src/EditableText/EditableText.test.js [hunk 4]
```diff
@@ -90,7 +90,7 @@ describe('InlineForm', () => {
 		expect(
 			wrapper
 				.find('Action')
-				.first()
+				.at(1)
```
Comment (jsomsanith) [inline packages/components/src/EditableText/EditableText.test.js]: This change seems curious

File: packages/components/src/EditableText/EditableText.scss (hunk 1 above)
Comment (ncomont) [reply packages/components/src/EditableText/EditableText.scss]: Without these props, buttons are not centered on IE.

File: packages/components/src/EditableText/EditableText.test.js [hunk 5]
```diff
@@ -77,7 +77,7 @@ describe('InlineForm', () => {
...
```
[the remark this answers is not in the document]
Comment (ncomont) [reply packages/components/src/EditableText/EditableText.test.js]: First is index 0. To follow the guidelines, I inverted the two buttons.

File: packages/components/src/EditableText/EditableText.test.js (hunk 4 above)
Comment (ncomont) [reply packages/components/src/EditableText/EditableText.test.js]: First is index 0. To follow the guidelines, I inverted the two buttons.

Comment (jdescottes-talend) [reply packages/components/src/EditableText/EditableText.test.js]: My brain was in a lua one based index, sorry --'

File: packages/components/src/EditableText/EditableText.scss [hunk 6]
```diff
@@ -69,25 +73,42 @@ $tc-input-editable-text-input-weight: $form-base-font-weight !default;
...
+			top: 0;
 
 			&-icon {
-				border: 0.2rem solid currentColor;
-				width: $tc-input-editable-text-size-large;
-				height: $tc-input-editable-text-size-large;
+				border: 0.1rem solid $dove-gray;
+				color: $dove-gray;
+				width: $tc-circle-editable-text-size;
+				height: $tc-circle-editable-text-size;
```
Comment (jdescottes-talend) [inline packages/components/src/EditableText/EditableText.scss]: the `margin-left: $padding-large;` seems to not be used anymore.
What could be done is replace it with the :last-child one `margin-left: $padding-small;` for the two here (and remove the one below)

File: packages/components/src/EditableText/EditableText.scss [hunk 7]
```diff
@@ -97,19 +118,29 @@ $tc-input-editable-text-input-weight: $form-base-font-weight !default;
...
+					height: $tc-icon-editable-text-size;
 				}
 
 				&:last-child {
 					margin-left: $padding-small;
 
 					> svg {
-						width: $tc-input-editable-text-size-small;
-						height: $tc-input-editable-text-size-small;
+						width: $tc-icon-editable-text-size;
```
Comment (jdescottes-talend) [inline packages/components/src/EditableText/EditableText.scss]: It's not useful anymore ?
It's the same as `> :global(.tc-svg-icon)` above

File: packages/components/src/EditableText/EditableText.scss (hunk 6 above)
Comment (ncomont) [reply packages/components/src/EditableText/EditableText.scss]: ✅

**Pull request merged**